ruby/red-arrow/ext/arrow/extconf.rb

```diff
@@ -42,6 +43,4 @@
   add_depend_package_path(name, source_dir, build_dir)
 end
 
-$CXXFLAGS += ' -std=c++11 -Wno-deprecated-register'
```

Comment:
@kou `-std=c++` is necessary to compile Apache Arrow on my development environment.

I created the related issue of extpp: https://github.com/red-data-tools/extpp/pull/5

Reply:
I've fixed and released Ext++ 0.0.5.

Reply:
Thanks!!